docs/guides/core-internals.mdx

```diff
@@ -4,12 +4,12 @@ description: A simplified version of the core implementation
 nav: 3.10
 ---
 
-This guide is beneficial for those who want to understand the core implementation of the jotai. It is not meant to be a complete guide of the 
-core implementation but rather a simplified version. It is inspired by the collection of tweets by Daishi Kato([@dai_shi](https://twitter.com/dai_shi)).
+This guide is beneficial for those who want to understand the core implementation of jotai. It's not meant to be a complete example of the 
+core implementation but rather a simplified version. It's inspired by the collection of tweets by Daishi Kato([@dai_shi](https://twitter.com/dai_shi)).
 
 ## First Version
-Let's start with the easy one. An atom is just a function that will return some config. We are using WeakMap to map atom with their state.
-WeakMap doesn't keep the keys in memory. So if atom is garbage collected, its state will be garbage collected too. It helps to avoid memory leaks.
+Let's start with an easy example. An atom is just a function that will return some configured value. We are using WeakMap to map atom with their state.
```

Comment:
It would be more accurate to say `configuration object` here rather than `configured value`.

Reply:
Sounds great! I've updated it. Thanks :smile: